woocommerce/woocommerce
# Implement the new subscriptions page design

### All Submissions:






### Changes proposed in this Pull Request:



This PR implements a new design for the **WooCommerce > Subscriptions** (experiment) page.

Before | After
--- | ---
![localhost_8888_wp-admin_admin php_page=wc-admin path=%2Fsubscriptions (1)](https://user-images.githubusercontent.com/1527164/166866431-04640832-c972-4959-92d4-fdca8881fa19.png) | ![localhost_8888_wp-admin_admin php_page=wc-admin path=%2Fsubscriptions](https://user-images.githubusercontent.com/1527164/166866442-ed40c723-28ba-4140-8d7b-2f4c89429627.png)

Please see #32691 for more information.

### How to test the changes in this Pull Request:

0. Checkout this branch and run `pnpm nx build-watch woocommerce-admin` to build assets
2. With WooCommerce Payments and WooCommerce Subscriptions inactive, navigate to **WooCommerce > Subscriptions**
3. Observe the new design. There shouldn't be any spelling mistakes or layout issues.
4. Clicking the "Get started" button should function as per before, i.e. it downloads and installs the WooCommerce Payments plugin.
5. The existing error handling should continue to work. See #32852 for testing instructions.

### Other information:




### FOR PR REVIEWER ONLY:
